src/component/short-cards.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Link from "./link.jsx";
+
+/**
+ * Nav renders a navigation resource.
+ *
+ * @param resource
+ */
+const ShortCards = ({ cards }) => {
+  // Get the current URL
+  const currentUrl = window.location.href;
+
+  return (
+    <div className="short-cards">
+      {cards.data.map(({ title, teaserIcon, teaserText, links }, i) => {
+        const linkHref = new URL(links.get('canonical').href, window.location.origin).href;
+        // Check if the resolved link matches the current URL
+        const isActive = linkHref === currentUrl;
```

Comment:
Oh the link in the json is supposed to have an `active` attribute, strange that it isn't there.
good workaround for now I guess but this should not be necessary